packages/deprecation-crawler/sandbox/deprecations/master.md

```diff
@@ -0,0 +1,98 @@
+# master (sandbox-date)
+
+## all-lowercase
+
+### crawled.ts
+
+#### lowercase1 (VariableStatement) {#3998890672}
+
+@deprecated internal use only
+  Example: all lowercase character
+  Group name: all lowercase
+  Regex: iNternal use onLy
+
+```ts
+const lowercase1 = 0;
+```
+
+#### lowercase2 (VariableStatement) {#2129363187}
+
+@deprecated Internal use only
+  Example: One uppercase character
+
+```ts
+const lowercase2 = 0;
+```
+
+#### lowercase3 (VariableStatement) {#2585813938}
+
+@deprecated inteRnal use oNly
+  Example: Multiple uppercase character
+
+```ts
+const lowercase3 = 0;
+```
+## catch-all
+
+### crawled.ts
+
+#### multiPatternMatch1 (VariableStatement) {#1734398741}
+
+@deprecated This const named `t` is deprecated. See {@link info} for xyz.
+  Example: 3 pattern match
+  Group name: multi pattern match
+  Regex: /^(?=.This const named!)(?=.deprecated!)(?=.xyz!).../
+
+```ts
+const multiPatternMatch1 = 0;
+```
+
+#### multiPatternMatch2 (VariableStatement) {#3477709142}
+
+@deprecated IMPORTANT! This const named `t1` is deprecated. See {@link info} for xyz.
+  Example: 3 pattern match with other start
+
+```ts
+const multiPatternMatch2 = 0;
+```
+
+#### multiPatternMatch3 (VariableStatement) {#2417065367}
+
+@deprecated This const named `t2` is deprecated. See {@link info} for xyz and related things.
+  Example: 3 pattern match with other end
+
+```ts
+const multiPatternMatch3 = 0;
+```
+## whitespace-normalisation
+
+### crawled.ts
```

Comment:
It's the file name.

In the test folder, we have different cases (e.g. `whitespace-normalisation`).
Each case has an `input` and an `output` file, before tests runs the `input` file is copied as the `crawled` file. Once this is done, we run the crawler against the sandbox project. In the tsconfig of the sandbox project we only include the copied `crawled` files. This means that the crawler will find for deprecations in the `crawled` files, and it will only modify them by adding the deprecation link.

```
{
  "include": ["./**/crawled.ts"]
}
```

After the crawling process we verify that the `crawled` files are equal to the `output` files.

The crawled files are exlcuded from git because these are basically our processed tests (and behave like a real repo).

Reply:
Got it!